frontend/dev.Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:dubnium
+
+WORKDIR /frontend
+
+COPY . .
```

Comment:
First copy package.json+lock, then install all deps, then copy rest of the code. Thanks to that all deps will be cached in the previous temp container layer and you get faster builds.